modules/button/react/lib/parts/ButtonContainer.tsx

```diff
@@ -49,13 +49,13 @@ function getIconColorSelectors(
 ): CSSObject {
   return {
     '&:focus span, &:hover span, & span': {
-      '.wd-icon-fill': {
+      '.wd-icon-fill, .wd-icon-accent, .wd-icon-accent2': {
         fill: color,
       },
       '.wd-icon-background': {
         fill: fill ? color : undefined,
       },
-      '.wd-icon-accent, .wd-icon-accent2': {
+      '.wd-icon-background ~ .wd-icon-accent, .wd-icon-background ~ .wd-icon-accent2': {
         fill: fill
           ? color === themePrimary.contrast
             ? themePrimary.main
```

Comment:
Prior to this PR, the coloring logic for accent layers (the nested ternary at the bottom of this code excerpt) only worked when toggling an IconButton on if the accent layer had a background layer behind it to provide contrast. It failed to account for the 50 or so icons which had accent layers but no background layer -- in these cases, the accent layers would be colored to `blueberry400` (assuming default Canvas theme) and blend right into the `blueberry400` of the toggled IconButton itself.

The trick was coloring the accent layers to the same color as the fill layer to start (`frenchVanilla100` in the default Canvas theme). And then only if there's a background layer, do we color the accent layers using the existing logic.

This solution is somewhat fragile in theory since it relies on `.wd-icon-background` being a _prior_ sibling of `.wd-icon-accent` and `.wd-icon-accent2`. I don't believe this should be an issue in practice, however, as I haven't encountered any icons where the background layer follows the accent layers.